javalib/src/main/scala/java/lang/Character.scala

```diff
@@ -1371,189 +1371,433 @@ object Character {
   }
 
   object UnicodeBlock {
-    val SURROGATES_AREA    = new UnicodeBlock("SURROGATES_AREA", 0x0, 0x0)
-    val BASIC_LATIN        = new UnicodeBlock("BASIC_LATIN", 0x0, 0x7f)
-    val LATIN_1_SUPPLEMENT = new UnicodeBlock("LATIN_1_SUPPLEMENT", 0x80, 0xff)
-    val LATIN_EXTENDED_A   = new UnicodeBlock("LATIN_EXTENDED_A", 0x100, 0x17f)
-    val LATIN_EXTENDED_B   = new UnicodeBlock("LATIN_EXTENDED_B", 0x180, 0x24f)
-    val IPA_EXTENSIONS     = new UnicodeBlock("IPA_EXTENSIONS", 0x250, 0x2af)
+    //////////////////
+    // Begin Generated
```

Comment:
I believe the software engineering would be improved by at least giving the 
reference Unicode version, the name and Git repository location of the script
used to generate the output, and the fact that it was manually folded (no build.sbt)
into the current file.
   
I do not see the usual "Do not edit" .

I also like to see who generated the output and date.  Some would say that this 
data can be more reliably figured out from GitBlame, but that is a recursive distraction from trying to read the code as is, on-the-fly.